packages/commonwealth/client/scripts/controllers/chain/substrate/shared.ts

```diff
@@ -225,7 +222,6 @@ class SubstrateChain implements IChainModule<SubstrateCoin, SubstrateAccount> {
 
   // load existing events and subscribe to future via client node connection
   public initChainEntities(): Promise<void> {
-    this._fetcher = new SubstrateEvents.StorageFetcher(this.api);
     const subscriber = new SubstrateEvents.Subscriber(this.api);
     const processor = new SubstrateEvents.Processor(this.api);
     return this._app.chain.chainEntities.subscribeEntities(
```

Comment:
This code listens for real-time entities from chain -- we may want to remove it.

Reply:
??? This code is already removed.

Reply:
You are correct that this listens for entities from the chain but we still need this code in order to listen to other events from the chain that concerns existing entities (i.e. votes being cast). So instead of removing this code I simply pushed a new change that edits the event handler function. Now the event handler augments entities with new events but the handler cannot create new entities.